src/components/dar_collection_table/DarCollectionTableCellData.js

```diff
@@ -56,8 +56,10 @@ const dacLinkToCollection = (darCode, status  = '', darCollectionId) => {
 };
 
```

Comment:
I can't put this comment in the right spot because it wasn't directly changed by this PR, but if I edit a draft and delete the project title, an empty string is displayed in the table instead of `- -`